extensions/amp-access/amp-access-spec.md

```diff
@@ -118,18 +117,18 @@ Here’s an example of the AMP Access configuration:
 
 ##Access URL Variables
 
-When configuring the URLs for various endpoints, the Publisher can use substitution variables. These variables are a subset of the variables defined in the [AMP Var Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md). The set of allowed variables is defined in the table below:
+When configuring the URLs for various endpoints, the Publisher can use substitution variables. These variables are a subset of the variables defined in the [AMP Var Spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md). The set of allowed variables is defined below.
```

Comment:
Actually https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md has the full list and the table is a mix of relevant fields and some additional substitutions